build/deploy/prometheus.libsonnet

```diff
@@ -126,7 +126,8 @@ local base = import 'base.libsonnet';
       },
     },
     service: base.Service(metadata, 'prometheus-service') {
-      metadata: {
+      app:: 'prometheus-service',
+      metadata+: {
```

Comment:
you're not adding anything new to metadata here.. let's delete the whole thing

Reply:
wait something was wrong, I definitely added it wrong here

Reply:
I have scrapping annotation below

Reply:
you don't need that, take a look at base.Service